### DISCUSSION TITLE
What does lobby.cs do?

### DISCUSSION THREAD

**Discussion by ivjvon** [winstxnhdw/lc-hax#286] [Q&A] [answered]
I notice that when I join a lobby from friends or public it says offline but when I join again it enters me, assuming it has to do with this file.

**Comment by winstxnhdw** [comment, marked as answer]
I am assuming you mean `LobbyPatch.cs`? It removes the server refresh delay and allows you to see lobbies with offensive names. The behaviour you describe is normal even without our mod.
